library/xwiki

```diff
@@ -1,18 +1,18 @@
-Maintainers: Vincent Massol <vincent@massol.net> (@vmassol)
+Maintainers: XWiki Core Dev Team <committers@xwiki.org> (@xwikiorg)
```

Comment:
I didn't want to hold up the update, so I merged but I have some questions about this change.

- What is "@xwikiorg"? It should be a user or group on GitHub, but it looks to be a twitter user.
- Is there a GitHub group that would allow us to see the XWiki Core Dev Team ("committers") to easily identify who can sign off on updates to the official image?

Reply:
Sorry it's a mistake, should be @xwiki as in https://github.com/xwiki/. Fixing that.